src/main/java/com/koreanlanguageschool/KoreanLanguageSchoolApplication.java

```diff
@@ -36,4 +35,13 @@ private static void getCourseClients() {
 
 		System.out.println(clientsByCourse);
 	}
+
+	private static void createClient(){
```

Comment:
let's rename this method - the name `create` indicates something would be returned but it's actually `void`... so I think a better name would be even `test`. Or, maybe even put the inside of this method into the `main` method and test the `operator` there :)

Reply:
Ok, done